packages/angular/carbon.yml

```diff
@@ -0,0 +1,275 @@
+  bar-floating:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#bar-(floating)
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/angular/?path=/story/simple-charts-bar-floating
+  bar-grouped:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#bar-(grouped)
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/angular/?path=/story/simple-charts-bar-grouped
+  bar-simple:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#bar-(simple)
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/angular/?path=/story/simple-charts-bar-simple
+  bar-stacked:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts/#bar-(stacked)
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/angular/?path=/story/simple-charts-bar-stacked
+  boxplot:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#boxplot
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/angular/?path=/story/simple-charts-boxplot
+  bubble:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#bubble
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/angular/?path=/story/simple-charts-bubble
+  bullet:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts/#bullet
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/angular/?path=/story/simple-charts-bullet
+  circle-pack:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/complex-charts#circle-packs
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/angular/?path=/story/complex-charts-circle-pack
+  combo:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/getting-started
```

Comment:
oh wow that's a gap, but yeah probably the best link for now

Reply:
Ya, I don't see any cds.com content for Combo, Sparkline, Step. In Carbon Next private beta, all assets need `externalDocsUrl`. So for now, we can point people to the cds.com data viz getting started. In a later platform release, we'll remove the `externalDocsUrl`s when we have detail pages populated.